src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckBomEditCollector.java

```diff
@@ -109,12 +106,15 @@ public BlackDuckBomEditCollector(JsonExtractor jsonExtractor, BlackDuckPropertie
                 List<LinkableItem> remediationItems = getBlackDuckDataHelper().getRemediationItems(componentVersionView);
                 componentAttributes.addAll(remediationItems);
 
-                ComponentItem.Builder builder = new ComponentItem.Builder();
-                builder.applyComponentData(componentItem)
-                    .applyAllComponentAttributes(componentAttributes)
-                    .applyCategory(BlackDuckVulnerabilityCollector.CATEGORY_TYPE)
-                    .applyOperation(ItemOperation.UPDATE)
-                    .applyNotificationId(notificationId);
+                ComponentItem.Builder builder = new ComponentItem.Builder()
+                                                    .applyCategory(BlackDuckVulnerabilityCollector.CATEGORY_TYPE)
+                                                    .applyOperation(ItemOperation.UPDATE)
+                                                    .applyComponentData(componentItem)
+                                                    // FIXME get the vulnerability id(s) and create a ComponentItem from each of them
```

Comment:
Right now BomEdit for vulnerabilities only reports the remediation information.  This fix me is to make vulnerability content the same as a vulnerability notification and include all of the current vulnerabilities?

Reply:
Correct. Did it ever include all the vulnerabilities?